user/how-to-guides/how-to-use-disposables.md

```diff
@@ -22,6 +22,22 @@ From inside an app qube, choosing the `Open in disposable` option on a file will
 
 This diagram provides a general example of how disposables can be used to safely open untrusted links and attachments in disposables. See [this article](https://blog.invisiblethings.org/2010/06/01/disposable-vms.html) for more on why one would want to use a disposable.
 
+## Named disposables and disposable templates
+
+There is a difference between [named disposables](/doc/glossary/#named-disposable) and [disposable templates](/doc/glossary/#disposable-template).
+
+In a default QubesOS Installation, you would probably use the 'whonix-ws-16-dvm' if you, as an example, want to browse the Tor network with an disposable. Every application starts a new random disposable with an ID in the name and if you close the window, it shuts down the qube. This is the feeling of an disposable template.
+
+In named disposables every application starts in the same qube, the qube itself has a fixed name and you need to manually shutdown the qube. Except from the non-persistance, they feel like usual app qubes. Named disposables are built upon disposable templates.
```

Comment:
```suggestion
Named disposable qubes function a bit differently - every application starts in the same qube, the qube itself has a fixed name and you need to manually shut it down. The only difference between a named disposable qube and a usual app qube is that a named disposable qube is not persistent - any changes are wiped when the qube is shut down. Named disposable qubes are, like normal disposable qubes, built upon disposable templates.
```

Reply:
Is "any changes are wiped when the qube is shut down" really the best wording?
I thought the changes are just gone but not explicitly overwritten.

Reply:
Yeah, this is a good point. Howeber, I think more explicitly describing it is a good idea? Maybe more like "changes are discarded"?